packages/git-mob-core/src/git-mob-api/git-authors/index.spec.js

```diff
@@ -50,6 +51,20 @@ test('.git-coauthors file does not exist', async () => {
   );
 });
 
+test('.git-coauthors by default is in the home directory', () => {
+  expect(pathToCoAuthors()).toEqual(path.join(process.env.HOME, '.git-coauthors'));
+});
+
+test('.git-coauthors can be overwritten by the env var', () => {
+  const oldEnv = process.env.GITMOB_COAUTHORS_PATH;
+  try {
+    process.env.GITMOB_COAUTHORS_PATH = '~/Env/Path/.git-co-authors';
+    expect(pathToCoAuthors()).toEqual('~/Env/Path/.git-co-authors');
+  } finally {
+    process.env.GITMOB_COAUTHORS_PATH = oldEnv;
+  }
+});
```

Comment:
@tlabeeuw could you add a test case for the local co-authors branch?

I think to mock the `fs.existsSync` function and make it return `true`.

Then the `pathToCoAuthors()` should return something like `./git-mob/.git-co-authors`. Maybe worth mocking the `topLevelDirectory()` as well. 

```
jest.spyOn(fs, 'existsSync').mockImplementation(() => true);
```

Reply:
Sounds good. Was also thinking instead of copy pasting the two pieces of code one should call the other. Which way round would be appropriate?